client/src/views/Parts.vue

```diff
@@ -37,7 +37,7 @@
               <a v-bind:href="'parts/' + row.partnumber">Edit</a>
             </td>
             <td>
-              <a @click="addtocart(row)">Add</a>
+              <a @click="addtocart({...row, qty: 1})">Add</a>
```

Comment:
I'm not sure if this fits in well with your overall architecture, but I needed a quantity field on the object. I don't know if you're familiar with the spread operator, but this is essentially a way of creating a new object with all the same fields as the row in addition to a new field for the quantity.